cocos/platform/mac/View.mm

```diff
@@ -49,15 +52,18 @@ - (instancetype)initWithFrame:(NSRect)frameRect {
         [self.window makeFirstResponder:self];
 
 #ifdef CC_USE_METAL
-        int pixelRatio = [[NSScreen mainScreen] backingScaleFactor];
-        CGSize size = CGSizeMake(frameRect.size.width * pixelRatio, frameRect.size.height * pixelRatio);
+        int    pixelRatio = [[NSScreen mainScreen] backingScaleFactor];
+        CGSize size       = CGSizeMake(frameRect.size.width * pixelRatio, frameRect.size.height * pixelRatio);
```

Comment:
As frameRect includes pixelRatio in Application-mac.mm in this commit. So do not have to multiply it again.

Reply:
pixelRatio in appdeligate.mm reverted, to keep logic size consistent with other platform.